packages/blade/src/_helpers/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Sandpack } from '@codesandbox/sandpack-react';
+import { DocsContext } from '@storybook/addon-docs';
+import dedent from 'dedent';
+import Box from '~components/Box';
+
+export type SandboxProps = {
+  children: string;
+  language?: 'js' | 'ts' | 'jsx' | 'tsx';
```

Comment:
do we need this type explicitly? we don't give the option to consumers in the playground right?

Reply:
I am setting `tsx` by default. If someone wants to add `ts` file, they can add it using this prop.

Reply:
By users here, I mean the developers integrating this reusable component.

So I can do-
```tsx
<Sandpack language="ts">
{`
  const a: number = 3;
  const b: string = 'hello';
  // ts code here
`}
</Sandpack>
```

Reply:
cool we can remove js and jsx in that case? because I don't see a need for that to be added in examples/

Reply:
yup makes sense

Reply:
done